### DISCUSSION TITLE
Where is the language menu file stored?

### DISCUSSION THREAD

**Discussion by psyca** [lxqt/lxqt#2428] [General]
Hi there,
i have a question whats the name or where the file is stored for the menu by default.
Iam talking about the menu where i made black lines.

![menux](https://github.com/lxqt/lxqt/assets/9515032/93909c85-6730-4ffa-bdda-ab2d86ddc7ea)

I have the problem, that they didnt showed as translated (english instead of german).

**Comment by stefonarch** [comment]
I updated the missing german translations recently but this will be available in LXQt 1.3 only (or using AUR git for arch & derivatives).

You can copy the relevant files from` /usr/share/desktop-directories/` to `~/.local/share/desktop-directories` ( "Other Settings" for example is `lxqt-settings-other.directory `)  and modify the [de] line.

"Other" and "System Tools" in your screenshot should be translated though, belonging to `lxmenu-data`. Which version do you use?

**Comment by psyca** [comment]
I found out that the distro uses the filename lxde-.... and some of these files are not filled. So i know now, whats the problem. Thanks for the info to look under /usr/share/desktop-directories. That helped me.
